hail/src/main/scala/is/hail/annotations/Region.scala

```diff
@@ -7,10 +7,15 @@ import is.hail.utils._
 import is.hail.nativecode._
 
 object Region {
-  def apply(sizeHint: Long = 128): Region = new Region()
+  val regular: Int = 64 * 1024
+  val small: Int = 8 * 1024
+  val tiny: Int = 256
 
-  def scoped[T](f: Region => T): T =
-    using(Region())(f)
+  def apply(blockSize: Int = regular): Region = new Region(blockSize)
```

Comment:
If the region implementation currently only works for these three sizes, this factory method (and the primary constructor) should probably be private.